training/network-partition-troubleshooting.md

```diff
@@ -15,3 +15,175 @@ In this lab, you'll... You'll then troubleshoot and resolve the problem.
 <div id="toc"></div>
 
 ## Before You Begin
+
+Note that this lab involves running a cluster in Docker so that you can use it to fake a partition between datacenters. You will need to have Docker Compose on your local machine, so you may just want to observe this one.
+
+## Step 1. Create a cluster in Docker across 3 simulated datacenters
+
+1. Download the Docker Compose file that defines a 6-node CockroachDB cluster spread across 3 separate networks:
+
+    <div class="filters clearfix">
+      <button style="width: 15%" class="filter-button" data-scope="mac">Mac</button>
+      <button style="width: 15%" class="filter-button" data-scope="linux">Linux</button>
+    </div>
+    <p></p>
+
+    <div class="filter-content" markdown="1" data-scope="mac">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ curl -O {{site.url}}/docs/training/resources/docker-compose.yaml
+    ~~~
+    </div>
+
+    <div class="filter-content" markdown="1" data-scope="linux">
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ wget {{site.url}}/docs/training/resources/docker-compose.yaml
+    ~~~
+    </div>
+
+2. Create the cluster:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ docker-compose up
+    ~~~~
+
+3. In a new terminal, initialize the cluster:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ docker exec -it roach-0 /cockroach/cockroach init --insecure
+    ~~~~
+
+4. Verify that the cluster is working by opening the Admin UI at <a href="http://localhost:8080" data-proofer-ignore>http://localhost:8080</a>.
+
+## Step 2. Create a partition in the network
+
+1. Disconnect the nodes in `dc-2` from the shared network backbone:
```

Comment:
nit: Since there's only one sub-step, you could remove the numbering. If you do so, you'll have to de-indent the code block.

Reply:
Left as is for now.